WORKSPACE

```diff
@@ -67,17 +69,17 @@ container_go_deps()
 
 # end external rules
 
-load("@io_bazel_rules_grafana//grafana:workspace.bzl", "repositories", "grafana_plugin")
+load("@io_bazel_rules_grafana//grafana:workspace.bzl", "grafana_plugin", "repositories")
 
-repositories()
+repositories(python_interpreter = "PY3")
```

Comment:
What happens if the user sets `PY3` here for the deps, but then creates a `PY2` dashboard?  is pip/grafanalib compatible with both versions?

Reply:
Most probably not. I can't think of a way to make it only one parameter for both py_dashboards() and repositories().

Reply:
hm.  How about remove this param, and have `repositories` create both the `pip_import` and `pip3_import` dependencies. Then in `py_dashboard`, choose `requirement` or `requirement3` based on the parameter.  Would that work?

Reply:
It will need to have both python and python3 binaries, but I think this is default on all major distributions.And it helps with keeping the number of params low.

Reply:
Yeah I think that's a safe assumption.

Reply:
done

Reply:
Not quite. WORKSPACE still needs:
```
load("@io_bazel_rules_grafana_deps//:requirements.bzl", pip_install_grafana = "pip_install")
load("@io_bazel_rules_grafana_deps3//:requirements.bzl", pip_install_grafana3 = "pip_install")

pip_install_grafana()
pip_install_grafana3()
```

Reply:
Any idea on how to make this more compact?

Reply:
That's fine, please update the README with that new call